src/main/java/com/sda/games/checkers/model/Spot.java

```diff
@@ -12,18 +12,137 @@
     private int y;
     private Piece piece;
 
+    public boolean isStartSpotValid(Board board, Player player, int startX, int startY) throws Exception {
+        if (player.isWhite()) {
```

Comment:
wow podziwiam ze Ci sie chcialo pisac tyle ifow :D 
jestem prawie pewien ze juz widzisz ze im wiecej ifow tym trudniej cos tu dopisac, i ze dalo by rade to przerobic ale to nie bedzie takie trywialne :P 
BTW w wolnej chwili mozesz looknac :P https://www.youtube.com/watch?v=xAMbxSCSyio